source/_integrations/rest.markdown

```diff
@@ -264,6 +268,24 @@ sensor:
 ```
 {% endraw %}
 
+[JSONPlaceholder](https://jsonplaceholder.typicode.com/) provides sample JSON data for testing. In the below example, jsonpath locates the attributes in the JSON document. [JSONPath Online Evaluator](https://jsonpath.com/) provides a tool to test your jsonpath. If the endpoint returns XML, it will be converted to JSON using xmltodict before searching for attributes.
+
+{% raw %}
+```yaml
+sensor:
+  - platform: rest
+    name: JSON users
+    json_attributes_path: "$.[0].address"
+    json_attributes:
+      - street
+      - suite
+      - city
+      - zipcode
+    resource: https://jsonplaceholder.typicode.com/users
+    value_template: '{{ value_json.name }}'
```

Comment:
Should be `{{ value_json[0].name }}`

Reply:
Thanks @arsaboo.  Will adjust this weekend as I'm traveling today.